### ISSUE TITLE
Player color page review

### ISSUE THREAD

**Issue by Zehir** [tts-community/Tabletop-Simulator-API#34]
After #24 and #23 what we do with the Player Color page ?
Any suggestions ?

**Comment by omniraptorr** [comment]
i still like the idea of just making it a table in types.md

**Comment by Zehir** [comment]
Yes we probably should fork the kb website too and have all in a sub domain. Like tts-api.ge.cx and tts-kb.ge.cx. And maybe an other main domain to be more findable on google.
